retinanet/model.py

```diff
@@ -259,23 +259,33 @@ def forward(self, inputs):
             transformed_anchors = self.regressBoxes(anchors, regression)
             transformed_anchors = self.clipBoxes(transformed_anchors, img_batch)
 
-            scores = torch.max(classification, dim=2, keepdim=True)[0]
+            finalResult = [[], [], []]
 
-            scores_over_thresh = (scores > 0.05)[0, :, 0]
+            finalScores = torch.Tensor([]).cuda()
```

Comment:
this will break inference when cuda isn't available, so you should check `torch.cuda.is_available()` before initializing the array